jscomp/bsb/bsb_ninja_check.ml

```diff
@@ -28,6 +28,7 @@ type t =
     dir_or_files : string array ;
     st_mtimes : float array;
     source_directory :  string ;    
+    cmdline_backend: Bsb_config_types.compilation_kind_t;
   }
 
```

Comment:
maybe we don't need add a new field, we can just change the name of file, for js backend its name is xx, for native backend its name is yy so they don't conflict with each other

Reply:
I am thinking to reduce such conflicts even for ninja file name, they can pick different names so less conflict is introduced, then you can call `ninja -f xx.ninja`